docker-entrypoint.sh

```diff
@@ -17,6 +17,4 @@ service postgresql restart
 su - postgres -c "psql iasql_metadata -c \"CREATE EXTENSION IF NOT EXISTS pg_cron;\""
 su - postgres -c "psql iasql_metadata -c \"GRANT EXECUTE ON FUNCTION cron.schedule_in_database(text,text,text,text,text,boolean) TO postgres;\""
 
-yarn forever -f dist/services/scheduler.js &
-yarn wait-on http://localhost:14527/health/
 yarn start
```

Comment:
Now `yarn start` would suffice to run the program. No need for running multiple processes for `iasql-engine` to start. That's a good thing.